tests/ci/component_governance.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Pull request against these branches will trigger this build
+pr: 
+  - master
+
+# no CI trigger
+trigger: none
+
+jobs:
+- job: Component_governance
+  timeoutInMinutes: 20 # how long to run the job before automatically cancelling
+  pool:
+    vmImage: 'ubuntu-16.04'
+
+  steps:
+  - bash: |
+      python scripts/generate_conda_file.py --gpu --pyspark
```

Comment:
using the conda file generator has the side-effect that you will only get requirements for the current OS. Right now using linux is fine because that doesn't exclude anything. But accessing the dicts in the generate_conda_file directly will provide the full list of options regardless of OS or flags used.

Reply:
I changed the functionality